lib/state/revisions/reducer.js

```diff
@@ -0,0 +1,16 @@
+import { combineReducers } from 'redux';
+
+const revisionState = ( state = { revision: null, isViewingRevisions: false }, action ) => {
+	if ( 'setRevisionState' !== action.type ) {
+		return state;
+	}
+	console.log( action );
+	return {
+		...state,
+		...action.revisionState,
+	};
+};
+
+export default combineReducers( {
+	revisionState,
+} );
```

Comment:
if only we could have had our Redux pow-wow last week! oh but this is a great little bit of code to illustrate some patterns. I'll come back in a bit with some more code to show what I mean, but `combineReducers` gives us the ease to eschew reducers with complex state

```js
export default combineReducers( {
	isViewingRevisions,
	revision,
} );
```